agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -298,6 +309,19 @@ private static ApplicationInsightsXmlConfiguration buildXmlConfiguration(Configu
         return xmlConfiguration;
     }
 
+    //assuming that every other element is not needed - such as "type", "worker", "name"
+    //assuming same set of delimiters in every object name
+    static String getJmxDisplayName(String attribute) {
+        String[] nameParts = attribute.split("=|:|,");
+        StringBuilder name = new StringBuilder();
+        if(nameParts.length>0) {
```

Comment:
I don't think you need this conditional, ok to loop 0 times below

Reply:
Changed